src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -1306,7 +1318,16 @@ public static class GroupRowFormat
             public static string Simulation = "Simulation";
             public static string Time = "Time";
             public static string Observation = "Observation";
-            public static string Undefined = "Undefined";
+            public static string IdentityCurve = "Identity";
+            public static string Undefined = "Undefined"; 
+         }
+
+         public static class DeviationLines
+         {
+            public static string SpecifyFoldValue = "Specify deviation fold value";
+            public static string DeviationLineDescription = "Will create two deviation lines according to the given fold value which has to be >= 1.\n" +
```

Comment:
hum. Can we not used + here again. Use the @ maybe or all in one line?

Reply:
I mean the main reason for me was readability here, since it is quite a long line. Still can easily let everything be in one line.